doc/HomePage.pod6

```diff
@@ -3,7 +3,7 @@
 Welcome to the official documentation of the <a href="https://raku.org">Raku</a>
 programming language!
 Besides online browsing and searching, you can also
-<del><a href="/perl6.html">view everything in one file</a> or</del>(See <a href="https://github.com/Raku/doc/issues/1981">Raku/doc#1981</a>)
+<a href="/perl6.html">view everything in one file</a> or
```

Comment:
While we're at this, we should probably call this page raku.html...

Reply:
> While we're at this, we should probably call this page raku.html...

Nice catch - I renamed it in the Makefile, but missed this one. Fixed in 54b36607d.